IdentityCore/src/MSIDOAuth2Constants.m

```diff
@@ -67,6 +67,8 @@
 NSString *const MSID_OAUTH2_PROMPT_NONE         = @"none";
 NSString *const MSID_OAUTH2_SIGNOUT_REDIRECT_URI    = @"post_logout_redirect_uri";
 NSString *const MSID_OAUTH2_REQUEST_CONFIRMATION = @"req_cnf";
+NSString *const MSID_OAUTH2_REQUEST_ENDPOINT = @"endpointUrl";
+NSString *const MSID_OAUTH2_HOME_ACCOUNT_ID = @"homeAccountId";
```

Comment:
I don't see MSID_OAUTH2_HOME_ACCOUNT_ID being used. Let's remove it.

Reply:
good catch. Thanks.